src/utils/browser-utils.js

```diff
@@ -4,7 +4,11 @@ export function getUrlSearchParams(locationSearch = window.location.search) {
   return locationSearch.substr(1).split('&').reduce(function (q, query) {
     const chunks = query.split('=');
     const key = chunks[0];
-    const value = chunks[1];
+    if(!chunks[1]) {
```

Comment:
These functions looks like something we can deprecate... 

`URLSearchParams` API only unavailable for IE-11 maybe we can get around with out it..

If not, we should use polyfills instead - its much more stable.

Reply:
Yeah. I've wrote those and I'm sure there are better ways to get browser url query parameters. probably using react route.

Reply:
I'm not actually using `URLSearchParams` - I believe I've had issues with mobile browsers - can't remember the issue back then - Anyway - Added a Polyfill cause it's 2019 (Hopefully done it the right way) - I also added a few other polyfills as I was freely using other cool latest ES features.
Will revert now to URLSearchParams cause it's probably more trusted then what I did there.

Reply:
Hmmm `URLSearchParams` is lame and can't handle delete for `label='my'&label='you` so I would need to keep some of my original functions